tests/test_admin.py

```diff
@@ -2828,3 +2835,41 @@ def test_edit_link_inactive(self):
         self.assertIn("inactive", response)
         self.assertIn('title="Edit"', response)
         self.assertNotIn(edit_endpoint, response)
+
+    def test_valid_back_link(self):
+        """Test if the discard view upon get request replaces the link for the back button with
```

Comment:
Test if is a redundant phrase that could be added to every test. 

```suggestion
        """The discard view upon get request replaces the link for the back button with
```